frontend/src/components/MovieList.tsx

```diff
@@ -0,0 +1,39 @@
+import {Movie} from "../Movie.ts";
+import styled from "styled-components";
+
+type MovieListProps = {
+    movies: Movie[]
+    title: string
+}
+
+export default function MovieList(props: MovieListProps) {
+
+    const posterImageUrl: string = "https://image.tmdb.org/t/p/w440_and_h660_face"
```

Comment:
You could think about moving it to a higher order and accept it as prop (in case it makes sense)

Reply:
For now, it's better to have the URL in the component to avoid prop drilling